core/modules/SConscript

```diff
@@ -194,8 +194,12 @@ shlibs = {}
 shlibs["qserv_common"] = dict(mods="""global memman proto mysql sql util""",
                               libs="""log protobuf mysqlclient_r boost_thread crypto""")
 
+# library implementing xrootd logging intercept (worker side)
+shlibs["xrdlog"] = dict(mods="""xrdlog""",
+                        libs="""log XrdSsiLib""")
+
 # library implementing xrootd services (worker side)
-shlibs["xrdsvc"] = dict(mods="""wbase wcontrol wconfig wdb wlog wpublish wsched xrdsvc""",
+shlibs["xrdsvc"] = dict(mods="""wbase wcontrol wconfig wdb wpublish wsched xrdsvc""",
```

Comment:
It looks like with you changes `wlog` will no longer be needed. Can you open a JIRA ticket to delete it at some point later so we don't forget to get rid of deadwood. Or maybe even drop it now if you feel confident.